src/components/pages/poll-page/components/answers-sheet/AnswersSheet.tsx

```diff
@@ -64,213 +37,216 @@ export const getProgress = (answers: Answer[], questions: Question[]) => {
-                      ) : question.type === 'TOGGLE' ? (
-                        <RadioGroup
-                          options={[
-                            { value: '1', label: 'так' },
-                            { value: '0', label: 'ні' },
-                          ]}
-                          sx={{
-                            display: 'flex',
-                            justifyContent: 'flex-start',
-                            gap: '36px',
-                          }} //TODO remove inline styles when refactor
-                          name={question.id}
-                        />
-                      ) : (
-                        <TextArea
-                          rowsNumber={numberRowsTextArea}
-                          sx={sxStyles.textArea}
-                          name={question.id}
-                        />
-                      )}
-
-                      {question.criteria && (
-                        <p className={styles['question-criteria']}>
-                          {question.criteria}
-                        </p>
-                      )}
-                    </div>
-                  ))}
-                  <Button
-                    className={styles['button']}
-                    text={
-                      isTheLast ? 'Завершити опитування' : 'Наступні питання'
-                    }
-                    type="submit"
-                    disabled={isTheLast && !isValid}
-                    onClick={async () => {
-                      answer(values);
-                      if (!isTheLast) {
-                        setCurrent(prev => ++prev);
-                        window.scrollTo({ top: 0, behavior: 'smooth' });
-                      } else {
-                        setIsSendingStatus(SendingStatus.LOADING);
-                        try {
-                          for (let i = 0; i < answers.length; i++) {
-                            if (i == answers.length - 1) {
-                              answers[i].value = answers[i].value
-                                .toString()
-                                .trim();
-                            }
-                            if (answers[i].value.length === 0) {
-                              answers = answers.filter(
-                                item => item !== answers[i],
-                              );
-                            }
-                            answers[i].value = answers[i].value.toString();
-                          }
-                          await PollAPI.createTeacherGrade(
-                            { answers },
-                            disciplineTeacherId,
-                          );
-                          setIsSendingStatus(SendingStatus.SUCCESS);
-                        } catch (error) {
-                          displayError(error);
-                          setIsSendingStatus(SendingStatus.ERROR);
-                        }
+                    className={styles['form']}
```

Comment:
as far as I know we agreed upon not using classNames and switched to `sx`, consider adding `Box` as a wrapper

Reply:
doesn`t work

Reply:
try one more time)
or screw it, I don`t care